hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/NoneRegionSplitPointRestriction.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.regionserver;
+
+import java.io.IOException;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.client.TableDescriptor;
+import org.apache.yetus.audience.InterfaceAudience;
+
+/**
+ * A RegionSplitPointRestriction implementation that does nothing.
+ */
+@InterfaceAudience.Private
+public class NoneRegionSplitPointRestriction extends RegionSplitPointRestriction {
```

Comment:
I'm not a native English speaker so I do not have better candidate but I feel the name is a bit strange...

@saintstack Do you have any suggestions on this? It should be NoRestriction I think?

Reply:
This is a bit odd, yeah.

So, first, why do I have a 'None' restriction? Why would I not just put the Restriction in place ?

On the name, NoRegion... would be better than NoneRegion... Or DoNothingRegion... would be best.